src/java.base/share/classes/java/security/InvalidParameterException.java

```diff
@@ -58,4 +58,44 @@ public InvalidParameterException() {
     public InvalidParameterException(String msg) {
         super(msg);
     }
+
+    /**
+     * Constructs an {@code InvalidParameterException} with the
+     * specified detail message and cause. A detail message is a {@code String} that describes
+     * this particular exception.
+     *
+     * <p>Note that the detail message associated with {@code cause} is
+     * <i>not</i> automatically incorporated in this exception's detail
+     * message.
+     *
+     * @param  msg the detail message (which is saved for later retrieval
+     *         by the {@link Throwable#getMessage()} method).
+     * @param  cause the cause (which is saved for later retrieval by the
+     *         {@link Throwable#getCause()} method).  (A {@code null} value
+     *         is permitted, and indicates that the cause is nonexistent or
+     *         unknown.)
+     *
+     * @since 20
+     */
+    public InvalidParameterException(String msg, Throwable cause) {
+        super(msg, cause);
+    }
+
+    /**
+     * Constructs an {@code InvalidParameterException} with the specified cause and a detail
```

Comment:
Keep these lines within 80 character width to be consistent with the rest of the file.